iac/terraform/aws/variables.tf

```diff
@@ -25,6 +25,16 @@ variable "budget_settings" {
   }
 }
 
+variable "domain_name" {
+  type    = string
+  default = "bloodconnect.net"
```

Comment:
Do not set a default value and clean your commits

Reply:
Vai, Should I pass the domain name through the pipeline or how can I pass?

Reply:
Using the pipeline. Check branch deployment pipeline

Reply:
this

Reply:
Pass the domain name through the pipeline.
You will find it with this [commit](https://github.com/CraftsmenLtd/BloodConnect/commit/d63606716de271322d56157a22e988a7556e82ca)